assembl/graphql/docstrings.py

```diff
@@ -144,6 +143,7 @@ class LegalContents:
     cookies_policy_attachments = Default.string_entry % ("Attachments for cookies policy.")
     privacy_policy_attachments = Default.string_entry % ("Attachments for privacy policy.")
     user_guidelines_attachments = Default.string_entry % ("Attachments for user guidelines.")
+    mandatory_legal_contents_validation = """A boolean flag to activate or not mandatory validation of legal contents after SSO login."""
```

Comment:
you can remove the `or not` here also :)

Reply:
True!